evaluate2.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
```

Comment:
Its a good practice to start with `set -euo pipefail` https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425?permalink_comment_id=3935570

Reply:
`set -u` caused an error with sdkman:

```
source ./prepare_spullara.sh
/Users/jason/.sdkman/bin/sdkman-init.sh: line 61: ZSH_VERSION: unbound variable
```

So we'll use `set -eo pipefail`

Reply:
This is due to https://github.com/sdkman/sdkman-cli/issues/597